### PR TITLE
Allow running the tests against existing deployment

### PR DESCRIPTION
**Pull request by msrb**

### CODE REVIEW

**Comment by msrb** [comment]
[test this]

**Comment by msrb** [comment]
cc @tuxdna :) Some very basic changes to make it possible to run the tests against OpenShift

File: integration-tests/runtest.sh [hunk 1]
```diff
@@ -3,7 +3,6 @@
 if [ ! -f /var/run/docker.pid ]
 then
     echo "!!! Docker service is probably not running !!!"
-    exit 1
```
Comment (tuxdna) [inline integration-tests/runtest.sh]: Not sure how this will affect the tests.

**Comment by tuxdna** [comment]
LGTM. Please check.

File: integration-tests/runtest.sh (hunk 1 above)
Comment (msrb) [inline integration-tests/runtest.sh]: if the docker is not running, then when running locally the tests will fail later a bit later, not immediately. with this change we can run the tests against openshift and docker doesn't have to be running locally.

Comment (tuxdna) [inline integration-tests/runtest.sh]: Make sense for OpenShift. Closing it.

**Pull request merged**